src/Components/People.js

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+import React, { Component } from 'react'
+
+export class People extends Component {
+    constructor (){
+        super();
+        this.state = {
+            people: [],
+            searchPerson: '',
+            personName: '',
+            personAge: '',
+            personGender: '',
+        }
+    }
+    peopleList = async () => {
+        const data = await axios.get("https://ghibliapi.herokuapp.com/people")
+        console.log(data.data)
+        this.setState({
+            people: data.data,
+        })
+    }
+    componentDidMount() {
+        this.peopleList();
+    }
+
+    handleChange = (e) =>{
+        
+        this.setState({
+            searchPerson: e.target.value
+        })
+    }
+
+    handleSubmit = async (e) => {
+        e.preventDefault()
+        const personData = this.state.people.filter(person => person.name === this.state.searchPerson);
+        console.log(personData)
+        if (personData.length === 0) {
+            this.setState({
+                searchPerson: '',
+                personName: '',
+                personAge: '',
+                personGender: '',
+            })
+            console.log(this.state.personName)
```

Comment:
this.setState is asynchronous, so this console.log may not report the updated state value.